src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -33,7 +33,7 @@ case class ApplyBinaryPrimOp(op: BinaryOp, l: IR, r: IR, var typ: Type = null) e
 case class ApplyUnaryPrimOp(op: UnaryOp, x: IR, var typ: Type = null) extends IR
 
 case class MakeArray(args: Array[IR], var typ: TArray = null) extends IR {
-  override def toString(): String = s"MakeArray(${args: IndexedSeq[IR]}, $typ)"
+  override def toString: String = s"MakeArray(${ args.toIndexedSeq }, $typ)"
```

Comment:
`.toIndexedSeq` allocates a Vector, which I guess doesn't much matter for printing the IR, but seems unnecessarily bad.